WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -124,6 +128,7 @@ class PageListViewModel @Inject constructor(
 
             pagesViewModel.pages.observeForever(pagesObserver)
             pagesViewModel.invalidateUploadStatus.observeForever(uploadStatusObserver)
+            pagesViewModel.authorSelectionUpdated.observeForever(authorSelectionChangedObserver)
```

Comment:
I guess this approach is fine, since it's already used in this class. However, in general I think we should consider `observeForever` as a last resort and "code smell".

Reply:
Agreed, kept with the model used in the class for this issue - managing the observe and then remove. But yes, observerForever is always active and probably not needed for any of these here.